container.go

```diff
@@ -77,7 +77,8 @@ type ImageBuildInfo interface {
 	GetDockerfile() string                          // the relative path to the Dockerfile, including the file itself
 	GetRepo() string                                // get repo label for image
 	GetTag() string                                 // get tag label for image
-	ShouldPrintBuildLog() bool                      // allow build log to be printed to stdout
+	ShouldPrintBuildLog() bool                      // allow build log to be printed
+	GetBuildLogWriter() io.Writer                   // for output of build log if ShouldPrintBuildLog is true
```

Comment:
Given this change is a breaking change in the sense that anybody implementing this interface will be broken (very improbably, but true), I thikn we could merge both methods into one: `LogWriter() io.Writer`, so consumers can choose to print to stdout, nil or a custom one.

Wdyt?

Reply:
Yes, I think it's really better to do this for the interface. I will make a commit with a fix.